common/supplemental/supplementalData.xml

```diff
@@ -3559,7 +3559,7 @@ XXX Code for transations where no currency is involved
 		<territory type="MY" gdp="933300000000" literacyPercent="93.1" population="31809700">	<!--Malaysia-->
 			<languagePopulation type="ms" populationPercent="75" officialStatus="official"/>	<!--Malay-->
 			<languagePopulation type="en" populationPercent="21"/>	<!--English-->
-			<languagePopulation type="zh_Hant" populationPercent="17"/>	<!--Chinese (Traditional)-->
+			<languagePopulation type="zh_Hans" populationPercent="17"/>	<!--Chinese (Traditional)-->
```

Comment:
Please change the comment to match. I would also add something like "See CLDR-13396 for Hans vs. Hant."

You can use git commit --amend to avoid adding a separate commit.